arch/x86_64/mcount-dynamic.c

```diff
@@ -486,12 +485,10 @@ static int patch_normal_func(struct mcount_dynamic_info *mdi, struct sym *sym,
 	memcpy(jmp_insn + JMP_INSN_SIZE, &jmp_target, sizeof(jmp_target));
 
 	if (info.has_jump)
-		orig = mcount_save_code(&info, jmp_insn, 0);
+		mcount_save_code(&info, jmp_insn, 0);
 	else
-		orig = mcount_save_code(&info, jmp_insn, sizeof(jmp_insn));
+		mcount_save_code(&info, jmp_insn, sizeof(jmp_insn));
 
-	/* make sure orig->addr same as when called from __dentry__ */
-	orig->addr += CALL_INSN_SIZE;
```

Comment:
Can we keep this address unchanged?  This is not a part of this change and I think we might use dynamic tracing for `-pg`..